web/beacon-app/src/features/topics/components/TopicQueryResult/ViewingEvent.tsx

```diff
@@ -29,7 +30,7 @@ const ViewingEvent = ({ totalResults, totalEvents, counter }: MetaDataProps) =>
         <p className="pt-2 font-semibold">
           <Trans>Meta Data</Trans>
         </p>
-        <MetaDataTable />
+        <MetaDataTable results={results} />
```

Comment:
it will be great to move this component to the parent but it's not urgent we can do it later.